packages/ef-runtime-client/src/ModuleLoader/ModuleLoader.ts

```diff
@@ -0,0 +1,29 @@
+export interface IModuleLoaderDependencies {
+  document: Document;
+  loaderSrc: string;
+}
+
+export class ModuleLoader {
+  private document: Document;
+  private loaderSrc: string;
+
+  constructor({ document, loaderSrc }: IModuleLoaderDependencies) {
+    this.document = document;
+    this.loaderSrc = loaderSrc;
+  }
+
+  async init(): Promise<void> {
```

Comment:
Again here do we need to have an `init` if we lready have a `constructor` where we can do the initiliazation.

Reply:
Lets have this discussion in the other comment.